src/Php73/Php73.php

```diff
@@ -21,16 +21,28 @@ final class Php73
 {
     public static $startAt = 1533462603;
 
+    private static $previousValue = "0";
+
     /**
      * @param bool $asNum
      *
      * @return array|float|int
      */
     public static function hrtime($asNum = false)
     {
-        $ns = microtime(false);
-        $s = substr($ns, 11) - self::$startAt;
-        $ns = 1E9 * (float) $ns;
+        $microtime = microtime(false);
+        $current = 1E9 * (float) $raw;
```

Comment:
What is `$raw`? Should it be `$microtime`?

Reply:
Ah, yeh. I changed my mind about the name of that variable. :P